pkgs/by-name/no/nohang/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  nixosTests,
+  fetchFromGitHub,
+  python3,
+  sudo,
+  libnotify,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nohang";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "hakavlad";
+    repo = "nohang";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-aRiBGnJHdtQ5euvJ4DSuCZO5i5OJVRiCesaatrIARmg=";
+  };
+
+  postPatch = ''
+    patchShebangs src
+    substituteInPlace src/* --replace 'notify-send' '${libnotify}/bin/notify-send' --replace 'sudo' '${sudo}/bin/sudo'
+  '';
+
+  buildInputs = [ python3 ];
```

If Python is used as a build tool and not a library, then it should be part of `nativeBuildInputs`

```suggestion
  nativeBuildInputs = [ python3 ];
```

---

I think the tool is just a [python script](https://github.com/hakavlad/nohang/blob/master/src/nohang) although it doesn't seem to have the .py extension...